content/quick-start/remote-development-using-a-remote-environment.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Remote Development: Using a Remote Environment"
+menu:
+  main:
+    parent: 'quick-start'
+    weight: 110
+
+weight: 110
+---
+After [creating a remote environment]({{< relref "remote-development-creating-a-remote-environment.md" >}}) you'll need to open the project directory in a terminal, then create or switch to the branch specified when generating the token. The `init` command can then be executed: 
```

Comment:
Actually it's not even done. Your current HEAD commit (whichever the branch you're in just right now) will be forced push to this `cpdev/...` branch by `cp-remote` without checking it out.